src/stages/stage3_events.c

```diff
@@ -741,939 +81,689 @@ Boss* stage3_spawn_scuttle(cmplx pos) {
 	return scuttle;
 }
 
-static Boss* stage3_create_midboss(void) {
-	Boss *scuttle = stage3_spawn_scuttle(VIEWPORT_W/2 - 200.0*I);
-
-	boss_add_attack(scuttle, AT_Move, "Introduction", 1, 0, scuttle_intro, NULL);
-	boss_add_attack(scuttle, AT_Normal, "Lethal Bite", 11, 20000, scuttle_lethbite, NULL);
-	boss_add_attack_from_info(scuttle, &stage3_spells.mid.deadly_dance, false);
-	boss_add_attack(scuttle, AT_Move, "Runaway", 2, 1, scuttle_outro, NULL);
-	scuttle->zoomcolor = *RGB(0.4, 0.1, 0.4);
-
-	boss_start_attack(scuttle, scuttle->attacks);
-	return scuttle;
+Boss* stage3_spawn_wriggle_ex(cmplx pos) {
+	Boss *wriggle = create_boss("Wriggle EX", "wriggleex", pos);
+	boss_set_portrait(wriggle, get_sprite("dialog/wriggle"), get_sprite("dialog/wriggle_face_proud"));
+	wriggle->glowcolor = *RGBA_MUL_ALPHA(0.2, 0.4, 0.5, 0.5);
+	wriggle->shadowcolor = *RGBA_MUL_ALPHA(0.4, 0.2, 0.6, 0.5);
+	return wriggle;
 }
 
-static void wriggle_slave_visual(Enemy *e, int time, bool render) {
```

Comment:
Removing stuff that has not yet been reimplemented is probably not a good idea. Makes it more difficult to continue porting.

Reply:
My terminally ADHD-addled brain likes having a cleanly-structured file with only the stuff that's actually properly implemented in it, especially for these sorts of porting tasks. Right now I'm looking at old vs new side-by-side and I find it more manageable.